scheduler_templates/slurm.sh

```diff
@@ -24,7 +24,7 @@
 #SBATCH --time=1-00:00:00
```

Comment:
@HendriceH better name this file after the machine to run on. Since we might have multiple slurm templates eg. for Horeka ....